lib/smart_answer_flows/overseas-passports/outcomes/getting_your_passport/_collect_in_person.govspeak.erb

```diff
@@ -0,0 +1,2 @@
+
+You must collect in person.  If you’re unable to, someone else can go on your behalf.
```

Comment:
@floehopper All of the content came from an external department and then was looked over by a member of the Content team before being added to the development backlog. I can only assume they came to an agreement about the language.